schema/settings.json

```diff
@@ -59,6 +59,12 @@
       "description": "Show the formatted date string in the given format. For example \"yyy-MM-dd HH:mm:ss\" results in \"2023-02-28 14:25:57 \", \"dd/MM/YYY HH:mm:ss (O)\" results in \"28/02/2023 14:25:57 (GMT +3)\". Find out more on unicode date field symbols on this link [https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table]",
       "default": "yyy-MM-dd HH:mm:ss",
       "pattern": "[yGuqQMLwdeEcabBhHkKmsSzOxX\\W].*"
+    },
+    "showOutputsPerSecond": {
+      "type": "boolean",
+      "title": "Show Outputs Per Second",
+      "description": "Show the outputs per second calculated from the last cell execution time and number of outputs.",
```

Comment:
It's not clear from the name if this is in real time. How about: "After a cell has finished running, show the outputs per second calculated from the last cell execution time and number of outputs."

Reply:
Sounds good. Updated the description with your copy.